README.md

```diff
@@ -85,6 +85,12 @@ An alternative to the Grunt build is provided via the `util.sh` shell script.
 1.  `mkdir google_closure_templates; cd google_closure_templates`
```

Comment:
Going to leave this as is until we figure out how to build soyutils.js  with Maven